OpenRA.Game/Player.cs

```diff
@@ -178,7 +178,7 @@ public Player(World world, Session.Client client, PlayerReference pr, MersenneTw
 			else
 			{
 				// Map player
-				ClientIndex = 0; // Owned by the host (TODO: fix this)
+				ClientIndex = world.LobbyInfo.Clients.FirstOrDefault(c => c.IsAdmin)?.Index ?? 0; // Owned by the host (TODO: fix this)
```

Comment:
Can/should we even fix that this player is owned by the host?

Reply:
That question opens a can of worms, since "the host" hasn't even existed as a concept for many years. If you're playing a game on a dedicated server that has two or more clients with `IsAdmin` set (the frontend UI prevents this, but #17659 would change that) then who should own this?

Reply:
#17659 simply re-using the `IsAdmin` flag doesn't sound like a good idea to me (otherwise this line needs to change again?). It would likely be better to have a new flag for that, or rather split into `(In)GameAdmin` and `LobbyAdmin`/`ServerAdmin`.

Reply:
Maybe. But then there's also the (good) idea to spread bot control over multiple clients to reduce performance overheads.